typescript/packages/subsurface-viewer/src/layers/wells/wellsLayer.stories.tsx

```diff
@@ -437,6 +437,70 @@ Wells3d.parameters = {
     },
 };
 
+export const VerticalWell = Template.bind({});
+VerticalWell.args = {
+    id: "well_176",
+    bounds: [-700, 500, 1400, 1700],
+    layers: [
+        new WellsLayer({
+            data: "./verticalwell.json",
+        }),
+    ],
+
+    views: {
+        layout: [1, 1],
+        viewports: [
+            {
+                id: "verticalwell",
+                show3D: true,
+            },
+        ],
+    },
+};
+
+VerticalWell.parameters = {
+    docs: {
+        description: {
+            story: "One single vertical well",
+        },
+        inlineStories: false,
+        iframeHeight: 500,
+    },
+};
+
+export const WellWithDuplicates = Template.bind({});
+WellWithDuplicates.args = {
+    id: "well_176",
+    bounds: [756125, 166200, 757327, 167938],
+    layers: [
+        new WellsLayer({
+            data: "./one_well_duplicate_vertices.json",
+            refine: true,
+            simplifiedRendering: true,
+        }),
+    ],
```

Comment:
I think it would be better to create some simple synthetic well trajectories in the stories themselves instead of adding more well files.